api/v1beta1/azurecluster_default.go

```diff
@@ -321,6 +310,45 @@ func (c *AzureCluster) setBastionDefaults() {
 	}
 }
 
+func (lb *LoadBalancerClassSpec) setAPIServerLBDefaults() {
+	if lb.Type == "" {
+		lb.Type = Public
+	}
+	if lb.SKU == "" {
+		lb.SKU = SKUStandard
+	}
+	if lb.IdleTimeoutInMinutes == nil {
+		lb.IdleTimeoutInMinutes = pointer.Int32Ptr(DefaultOutboundRuleIdleTimeoutInMinutes)
+	}
+}
+
+func (lb *LoadBalancerClassSpec) setNodeOutboundLBDefaults() {
+	lb.Type = Public
+	lb.SKU = SKUStandard
+	if lb.IdleTimeoutInMinutes == nil {
+		lb.IdleTimeoutInMinutes = pointer.Int32Ptr(DefaultOutboundRuleIdleTimeoutInMinutes)
+	}
+}
+
+func setControlPlaneOutboundLBDefaults(lb *LoadBalancerClassSpec, apiserverLBType LBType) {
```

Comment:
I think we can combine outboundLB defaults into one method if the above refactoring is done.